Python/To-do List/README.md

```diff
@@ -0,0 +1,40 @@
+## README.md
+
+**Description:**
+
+With the rapid development of the world, everyone is engaged in a lot of work or even stuck with
+
+the thought of where to start from. To resolve this customary issue here presents ‘ON IT!!’.It is a
+
+python based GUI that helps boost productivity and helps get our work done in an organized
+
+way. It includes 4 fields to note the tasks to be performed today, tomorrow, the current week, or
+
+----------------------------------------------------------------------------------------------------------------------
+**Tech-Stack:**
+
+1.Python
+2.Tkinter
+
+---------------------------------------------------------------------------------------------------------------------------
+**Video:**
+
+[Video Link](https://drive.google.com/file/d/19s8E6qCUjR9Du1qNaTrbOPS3vwytF1es/view?usp=sharing)
+
+-----------------------------------------------------------------------------------------------------------------------------
+**Basic Components:**
+
+**Python**-Loops,Functions,etc
+
+**Tkinter** -PhotoImage, Label-Frames, Entry box, Button, icon bitmap
+
+
+---------------------------------------------------------------------------------------------------------------------------------
+**Steps to Run in Local:**
+1. Download the folder name ‘To-do list’
+
+2. Open the folder in any IDE which supports python (used Version- Python 3.10,IDE-Pycharm)
+
+3. Install pillow library by running the command- pip install pillow in the terminal
+
+4. Run the program

```

Comment:
mention it in these manner this part 
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/157317777-c7776579-dadb-4b8c-a8f6-e828d692d94d.png">

mention install Dependencies in the Readme file

<img width="1100" alt="Screenshot 2022-03-03 at 1 02 07 PM" src="https://user-images.githubusercontent.com/78725162/157317794-32661fe7-7c75-4b1e-9547-c2313f1edf51.png">